src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -3913,6 +3913,380 @@ class StubGenerator: public StubCodeGenerator {
     return start;
   }
 
+  // Set of L registers that correspond to a contiguous memory area.
+  // Each 64-byte register typically corresponds to 2 32-byte integers.
```

Comment:
`64-byte` -> `64bits`, same for `32-byte`.

Reply:
64-byte register! Will fix.